python/argo_client/connection.py

```diff
@@ -69,6 +72,22 @@ def get_one_reply(self) -> Optional[str]: pass
     @abstractmethod
     def send_one_message(self, the_message: str, *, expecting_response : bool = True) -> None: pass
 
+    def logging(self, on : bool, *, dest : TextIO = sys.stderr) -> None:
```

Comment:
I'm not sure if this is intentional, but there is a widely used [`logging` module](https://docs.python.org/3/library/logging.html) in Python that this method could be confused for.

Reply:
Given that it's an instance method, so it will always be proceeded by `OBJECT.` for some `OBJECT`, I'm inclined to think it won't be confused with that module, however I'm more than happy to reconsider/change it if others disagree and prefer a different name.

Reply:
I don't have a strong opinion on the matter—I just wanted wanted to the resemblance was noted. I'm quite fine with bucking convention :)